homeassistant/components/select/strings.json

```diff
@@ -28,6 +28,9 @@
       "state_attributes": {
         "options": {
           "name": "Options"
+        },
+        "option": {
+          "name": "Option"
```

Comment:
This is somewhat funky. The select entity does not actually have an option attribute. But it does have option attribute for when service calls change it's value, and it does have an "current_option" property. So it's somewhat mix and match on naming here.

Ideas welcome.